automated/linux/ota-update/download-update.sh

```diff
@@ -73,7 +79,12 @@ cp aklite-callback.sh /var/sota/
 chmod 755 /var/sota/aklite-callback.sh
 
 mkdir -p /etc/sota/conf.d
-cp z-99-aklite-callback.toml /etc/sota/conf.d/
+mkdir -p /etc/sota/conf.d
```

Comment:
copy-paste from ota-rollback I guess, one line too much.

Would it make sense to harmonise those two tests? since a lot of the code are the same...

Reply:
I tried, but it's harder than it seems. There are a lot of small differences. Fixed this issue